karma.conf.js

```diff
@@ -0,0 +1,49 @@
+const packageJSON = require('./package.json')
+
+module.exports = (config) => {
+	const karmaConfig = {
```

Comment:
I will later refactor this into a version2 of the resin-config-karma module, for now it's just faster to experiment here.